apps/sms/views/shared/test/unit/sms_test.js

```diff
@@ -114,7 +114,7 @@ suite('SMS App Unit-Test', function() {
     return elements;
   }
 
-  function threadSetupAppend() {
+  function threadSetup() {
```

Comment:
I admit the the unit test here looks not really qualified :/ ... But I simply make the test runnable instead of rewriting this because we'll going to refactor this later.